exporters/stdout/stdouttrace/trace.go

```diff
@@ -83,6 +84,8 @@ func (e *Exporter) ExportSpans(ctx context.Context, spans []trace.ReadOnlySpan)
 		}
 
 		// Encode span stubs, one by one
+		e.encoderMu.Lock()
```

Comment:
I think this needs to move outside the loop at 74.  This will work fine with the simple span processor, which only sends `spans` with length 1, but will deadlock with the batch span processor when it tries to take a lock when processing the second span.

Reply:
will fix!

Reply:
I'll fix!